src/main/java/org/folio/oaipmh/helpers/AbstractHelper.java

```diff
@@ -419,27 +434,20 @@ protected HeaderType addHeader(String identifierPrefix, Request request, JsonObj
 
   /**
    * Checks if request sequences can be resumed without losing records in case of partitioning the whole result set.
-   * <br/>
-   * The following state is an indicator that flow cannot be safely resumed:
-   * <li>No instances are returned</li>
-   * <li>Current total number of records is less than the previous one and the first
-   * record id does not match one stored in the resumptionToken</li>
-   * <br/>
-   * See <a href="https://issues.folio.org/browse/MODOAIPMH-10">MODOAIPMH-10</a> for more details.
    *
-   * @param request
-   * @param totalRecords
-   * @param instances
-   * @return
+   * @param request      oai-pmh request
+   * @param totalRecords records count into the system
+   * @param instances    instances from srs
+   * @return true if resumptionToken valid and request sequence can continue
    */
   protected boolean canResumeRequestSequence(Request request, Integer totalRecords, JsonArray instances) {
     Integer prevTotalRecords = request.getTotalRecords();
     boolean isDeletedRecords = isDeletedRecordsEnabled(request);
     int firstPosition = 0;
     return instances != null && instances.size() > 0
-        && (totalRecords >= prevTotalRecords || StringUtils.equals(request.getNextRecordId(),
-            isDeletedRecords ? storageHelper.getId(instances.getJsonObject(firstPosition))
-                : storageHelper.getRecordId(instances.getJsonObject(firstPosition))));
+      && (totalRecords >= prevTotalRecords || StringUtils.equals(request.getNextRecordId(),
+      isDeletedRecords ? storageHelper.getId(instances.getJsonObject(firstPosition))
```

Comment:
usage of deprecated storageHelper.getId method?

Reply:
I don't think that this method is deprecated. It would be better to remove the @Deprecated annotation from such method. It is used and works well. I'll remove the annotation.